README.md

```diff
@@ -51,6 +52,13 @@ features or security checks, please open an issue!
 - [Umbrella Apps](#umbrella-apps)
 - [Updates](#updates)
 
+## Documentation
```

Comment:
This section should be moved below Installation section to reflect table of contents